src/main/java/com/nyxei/pollingus/domain/messages/Component.java

```diff
@@ -0,0 +1,83 @@
+package com.nyxei.pollingus.domain.messages;
```

Comment:
Is `Component` an entity or a message ?
Is `Component` an anonymous item or should you rename it before its real identity ?

Reply:
Component is an entity that will allow to form a message